packages/map/src/test/generateRuntimeLegendHash.test.ts

```diff
@@ -0,0 +1,907 @@
+          Rate: 57,
+          Location: 'Home',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Northern Mariana Islands',
+          Rate: '60',
+          Location: 'Home',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Ohio',
+          Rate: 88,
+          Location: 'Vehicle',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Oklahoma',
+          Rate: 40,
+          Location: 'Work',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Oregon',
+          Rate: 55,
+          Location: 'Home',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Palau',
+          Rate: 15,
+          Location: 'School',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Pennsylvania',
+          Rate: 60,
+          Location: 'School',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Puerto Rico',
+          Rate: 30,
+          Location: 'Work',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Rhode Island',
+          Rate: 40,
+          Location: 'Vehicle',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'South Carolina',
+          Rate: 55,
+          Location: 'Home',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'South Dakota',
+          Rate: 86,
+          Location: 'Home',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Tennessee',
+          Rate: 60,
+          Location: 'Home',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Texas',
+          Rate: 30,
+          Location: 'Vehicle',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Utah',
+          Rate: 54,
+          Location: 'Work',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Vermont',
+          Rate: 40,
+          Location: 'Home',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Virgin Islands',
+          Rate: 55,
+          Location: 'School',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Virginia',
+          Rate: 57,
+          Location: 'School',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Washington',
+          Rate: 62,
+          Location: 'Work',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'West Virginia',
+          Rate: 25,
+          Location: 'Vehicle',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Wyoming',
+          Rate: 43,
+          Location: 'Vehicle',
+          URL: 'https://www.cdc.gov/'
+        }
+      ],
+      dataFileName: 'valid-data-map.csv',
+      dataFileSourceType: 'file',
+      formattedData: [
```

Comment:
can the data be refactored to use fakerjs or to pull the data from some example file

Reply:
We could use fakerjs, I thought testing a real example after using console logs to get a real hash might be beneficial here to grab the actual hash created. I'm not sure how important that actually is, but we might be able to identify when a real hash has changed/needs to be updated.

Reply:
Used example in examples folder for now.